lua/nvim-tree/actions/fs/remove-file.lua

```diff
@@ -9,7 +9,7 @@ local M = {
 }
 
 local function close_windows(windows)
-  if view.View.float.enable and #vim.api.nvim_list_wins() == 1 then
+  if view.View.float.enable and #vim.api.nvim_list_wins() < 3 then
```

Comment:
Please leave a comment explaining the magic number 3

Reply:
Initial problem was in case where list of wins is 2 (float tree, and opened buffer), so when trying to delete opened buffer window, vim throws an error cannot close last window. Is there a case where list can be 1? If it possible we save previous condition and add `if not 2` which prevents from closing the last window.

Reply:
Additionally we could check if the window count is 2, and there is no floating tree, in which case we could close the window. But even with `quit_on_focus_loss`, `list_wins` length would be 2 including floating tree.

Reply:
Safety first, thank you.